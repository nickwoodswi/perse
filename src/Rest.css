.set-rest{
    width: 100%;
}