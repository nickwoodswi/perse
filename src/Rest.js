import React, { Component } from 'react'

class Rest extends Component {
    render() {
        if (!this.props.rest) {
            return(
                <></>
            )
        }
        else {
            return(
                <> ({this.props.rest}sec rest)</>
            )
        }
    }
}

export default Rest