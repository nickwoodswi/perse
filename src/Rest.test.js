import React from 'react';
import { render } from '@testing-library/react';
import Rest from './Rest';
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

test('renders Rest component', () => {
    render(<MemoryRouter><Rest /></MemoryRouter>);
});

it('Rest component renders the UI as expected', () => {
    const tree = renderer
        .create(<MemoryRouter><Rest /></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});