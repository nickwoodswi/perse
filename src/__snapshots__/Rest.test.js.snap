// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rest component renders the UI as expected 1`] = `
<div
  className="set-rest"
>
  <i>
    (No rest after set)
  </i>
</div>
`;
